tools/monorepo-utils/src/code-freeze/commands/changelog/lib/index.ts

```diff
@@ -0,0 +1,187 @@
+/**
+ * External dependencies
+ */
+import simpleGit from 'simple-git';
+import { execSync } from 'child_process';
+
+/**
+ * Internal dependencies
+ */
+import { Logger } from '../../../../core/logger';
+import { checkoutRemoteBranch } from '../../../../core/git';
+import { createPullRequest } from '../../../../core/github/repo';
+import { Options } from '../types';
+
+/**
+ * Perform changelog operations on release branch by submitting a pull request. The release branch is a remote branch.
+ *
+ * @param {Object} options       CLI options
+ * @param {string} tmpRepoPath   temp repo path
+ * @param {string} releaseBranch release branch name. The release branch is a remote branch on Github.
+ * @return {Object} update data
+ */
+export const updateReleaseBranchChangelogs = async (
+	options: Options,
+	tmpRepoPath: string,
+	releaseBranch: string
+): Promise< { deletionCommitHash: string; prNumber: number } > => {
+	const { owner, name, version } = options;
+	try {
+		await checkoutRemoteBranch( tmpRepoPath, releaseBranch );
+	} catch ( e ) {
+		if ( e.message.includes( "couldn't find remote ref" ) ) {
+			Logger.warn(
```

Comment:
I figured this would be a common error, so wanted to provide more context than just `couldn't find remote ref`. If we want one or the other, I'd say `release/7.8 does not exist on obliviousharmony/woocommerce` is more informative.

Fixed in https://github.com/woocommerce/woocommerce/pull/38082/commits/664d90acf8c4f30e53b5481ffdaf3f21381e6f3a